insights/parsers/rhc_conf.py

```diff
@@ -0,0 +1,49 @@
+"""
+RHCConf - file ``/etc/rhc/config.toml``
+=======================================
+"""
+
+from insights import Parser, parser
+from insights.parsers import get_active_lines
+from insights.specs import Specs
+from insights.core.dr import SkipComponent
+
+
+@parser(Specs.rhc_conf)
+class RHCConf(Parser, dict):
```

Comment:
A nitpick, but would you think about renaming it to `RhcConf` a CamelCase naming style?

Reply:
@xiangce RHC is an abbreviation of "Remote Host Connection", so I used uppercase for all of them. But it's good to me to update it to "RhcConf" to avoid two uppercase "C" in the middle, I have updated it. And it seems there is only one configuration file for rhc client, so I just use "RhcConf" without "Toml". Please help to review again, thanks.